website/docs/d/wafregional_rate_based_rule.html.markdown

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Can you please add a sidebar link to this page in the `website/aws.erb` file? Thanks!